versioned_docs/version-v1.3/rancher/csi-driver.md

```diff
@@ -279,5 +279,21 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
     - If you leave the `Host StorageClass` field empty, the default StorageClass of the Harvester cluster will be used.
 
     :::
-  
-1. You can now create a PVC based on this new **StorageClass**, which utilizes the **Host StorageClass** to provision volumes on the bare-metal Harvester cluster.
\ No newline at end of file
+
+1. You can now create a PVC based on this new **StorageClass**, which utilizes the **Host StorageClass** to provision volumes on the bare-metal Harvester cluster.
+
+## Upgrade CSI Driver
+
+### Upgrade RKE2
+The csi driver can be upgraded by upgrading the RKE2 version. You can upgrade the RKE2 cluster via the Rancher UI as follows:
```

Comment:
Looks like, the statement is missed to be modified at the version level doc.
```
To upgrade the CSI driver, use the Rancher UI to upgrade RKE2. Ensure the new RKE2 version supports/bundled with the updated CSI driver version.
```

Reply:
Thank you. Updated it.